ui/rangeslider/rangeslider.android.ts

```diff
@@ -0,0 +1,183 @@
+/*globals requireClass*/
+import AndroidConfig from '../../util/Android/androidconfig';
+import AndroidUnitConverter from '../../util/Android/unitconverter.js';
+import { RangeSliderEvents } from './rangeslider-events';
+import { ViewAndroid } from '../view/view.android';
+import { AndroidProps, IRangeSlider } from '.';
+import Color from '../color';
+import { eventCallbacksAssign } from '../../core/eventemitter/eventCallbacksAssign';
+
+const NativeSFRangeSlider = requireClass('io.smartface.android.sfcore.ui.rangeslider.SFRangeSlider');
+
+export default class RangeSliderAndroid<TEvent extends string = RangeSliderEvents> extends ViewAndroid<TEvent | RangeSliderEvents, AndroidProps> implements IRangeSlider {
+  private _snapStepSize: number = 1;
+  private _minValue: number = 0;
+  private _maxValue: number = 5;
+  private _thumbBorderWidth: number;
+  private _trackWeight: number;
+  private _outerTrackWeight: number;
+  private _trackColor: Color;
+  private _outerTrackColor: Color;
+  private _thumbColor: Color;
+  private _thumbBorderColor: Color;
+  private _thumbSize: number = 5;
+  private _isTrackRounded: boolean = true;
+  private _rangeEnabled: boolean = true;
+  private _maxValueChanged: boolean = false;
+  private _onValueChange: (value: number[]) => void;
+
+  constructor(params: Partial<IRangeSlider> = {}) {
+    super();
+
+    if (!this.nativeObject) {
+      this._nativeObject = new NativeSFRangeSlider(AndroidConfig.activity);
+    }
+
+    const self = this;
+    const android = {
+      get thumbSize(): number {
+        return self._thumbSize;
+      },
+      set thumbSize(value: number) {
+        self._thumbSize = value;
+        self.nativeObject.setThumbSize(AndroidUnitConverter.dpToPixel(self._thumbSize));
+      },
+      get thumbColor(): Color {
+        return self._thumbColor;
+      },
+      set thumbColor(value: Color) {
+        self._thumbColor = value;
+        self.nativeObject.setThumbColor(self._thumbColor.nativeObject);
+      },
+      get thumbBorderColor(): Color {
+        return self._thumbBorderColor;
+      },
+      set thumbBorderColor(value: Color) {
+        self._thumbBorderColor = value;
+        self.nativeObject.setThumbBoundaryColor(self._thumbBorderColor.nativeObject);
+      },
+      get thumbBorderWidth(): number {
+        return self._thumbBorderWidth;
+      },
+      set thumbBorderWidth(value: number) {
+        self._thumbBorderWidth = value;
+        self.nativeObject.setThumbBoundarySize(AndroidUnitConverter.dpToPixel(self._thumbBorderWidth));
+      },
+      get outerTrackWeight(): number {
+        return self._outerTrackWeight;
+      },
+      set outerTrackWeight(value: number) {
+        self._outerTrackWeight = value;
+        self.nativeObject.setBarWeight(AndroidUnitConverter.dpToPixel(self._outerTrackWeight));
+      }
+    };
+    Object.assign(this._android, android);
+
```

Comment:
`this.nativeObject.setOnValueChange(() => {
  this.emit(RangeSliderEvents.ValueChange);
  this.onValueChange?.(e);
})`
ekleyelim